src/crawl/DeepCrawler/DeepCrawler.js

```diff
@@ -0,0 +1,161 @@
+import * as cheerio from 'cheerio';
+
+import { BaseCrawler } from '../BaseCrawler.js';
+import { SCRAPE_PAGE_PROMPT, SCRAPE_TYPE_PROMPT, ANALYZE_PAGE_PROMPT, GENERATE_SCHEMA_PROMPT } from './prompts.js';
```

Comment:
lower case, also I thikn we should start doing

```
import { * as prompts } from './prompts.js'
```
and then reference it as `prompts.scrapePage` etc.,